BioscoopSysteemWeb/BioscoopSysteemWeb/Service/MovieService.cs

```diff
@@ -20,19 +20,19 @@ public async Task<IEnumerable<MovieReadDTO>> GetMovies()
         {
             try
             {
-                var respone = await _httpClient.GetAsync("api/movies");
+                var respose = await _httpClient.GetAsync("api/movies");
```

Comment:
Nu denk je een typo te hebben gefixed maar blijkt het toch niet zo te zijn :p 

Response moet het zijn toch?

Reply:
Haha klopt. Ik had hem ergens anders al gefixed, alleen hier was ik het vergeten😉

Reply:
Fixed in de laatste commit